internal/backend/crypto/age/ssh.go

```diff
@@ -32,14 +32,11 @@ func (a *Age) getSSHIdentities(ctx context.Context) (map[string]age.Identity, er
 		return sshCache, nil
 	}
 
-	// notice that this respects the GOPASS_HOMEDIR env variable, and won't
-	// find a .ssh folder in your home directory if you set GOPASS_HOMEDIR
-	uhd := appdir.UserHome()
-	sshDir := filepath.Join(uhd, ".ssh")
-	if !fsutil.IsDir(sshDir) {
-		debug.Log("no .ssh directory found at %s. Ignoring SSH identities", sshDir)
+	sshDir, err := getSSHDir()
+	if err != nil {
+		debug.Log("asdf: %s", err)
 
-		return nil, fmt.Errorf("no identities found: %w", ErrNoSSHDir)
+		return nil, fmt.Errorf("asdf: %w", err)
```

Comment:
That's a weird error?

Reply:
sorry it was just a placeholder. wanted to get your early opinion